msg/en_US/utils.msg

```diff
@@ -986,6 +986,7 @@ valid options:\n\
   -p, --page=ID               page id; the -L option is required; default : 0\n\
   -v, --verbose               enable verbose status messages; default : disable\n\
   -i, --interval=S            print information every S secs\n
+117 Query result is failed to load. First, wrong directory of log file makes this error. Second, deletion db_ha_apply_info infomation makes this error. \n
```

Comment:
I think it should inform when this error occurs, and I guess users don't know why this is related to the "query result". Plus, you can make the error case manually by deleting a record in the catalog class, but it would occur whenever the table has no proper data for some reason. With this, my trial to make the sentence natural (just a suggestion):
```suggestion
117 Can't generate the applied info due to a wrong location of the log volume or a problem in db_ha_apply_info catalog table. \n
```

Reply:
Thank you. I think your suggestion is more appropriate than my error message!